src/plugins/intel_gpu/src/kernel_selector/cl_kernels/softmax_gpu_bf.cl

```diff
@@ -304,11 +321,210 @@ KERNEL (softmax_gpu_continuous_bfyx)(
         output[aligned_data_offset + get_sub_group_local_id() + output_idx * get_sub_group_size()] = ACTIVATION(my_chunk[output_idx] / my_sum, ACTIVATION_PARAMS);
     }
 
-    if (in_data_set_idx < aligned_offset)
+    if (in_data_set_idx < aligned_offset) {
         output[data_set_offset + in_data_set_idx] = ACTIVATION(my_chunk[output_idx++] / my_sum, ACTIVATION_PARAMS);
+    }
 
-    if (in_data_set_idx < actual_leftovers)
+    if (in_data_set_idx < actual_leftovers) {
         output[leftover_idx] = ACTIVATION(my_chunk[output_idx++] / my_sum, ACTIVATION_PARAMS);
+    }
+#endif
+#if IS_DYNAMIC
+    } else { // Case for output[] directly when (items_num + 2) > STACK_SIZE
```

Comment:
random spot. 
Looks that we have too much duplicate codes. 
I think we can simplify it like 
- tmp_out_ptr = &my_chunk[0]; if use_slm 
- otherwise tmp_out_ptr = output;

And then, change the strides according to the cases.

Reply:
@yeonbok Implemented tmp_out_ptr with stride calculation function. Please let me know whether it looks ok or not. If ok, I will check performance impact.